app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -4012,6 +4012,10 @@ public boolean isPlaying() {
         return !exoPlayerIsNull() && simpleExoPlayer.isPlaying();
     }
 
+    public boolean isPlayingWhenReady() {
```

Comment:
Rename this to `getPlayWhenReady` and check if it can be used in other places (i.e. in other places where currently `simpleExoPlayer.getPlayWhenReady()` is being used.
```suggestion
    public boolean getPlayWhenReady() {
```